resources/styles/components/course-listings.less

```diff
@@ -49,18 +49,14 @@
     }
 
     &[data-category=physics] {
-      .tutor-course-item {
-        .tutor-physics-cover-background();
-      }
+      &[data-title=Physics]{ .tutor-course-item { .tutor-physics-cover-background(); } }
```

Comment:
No that's targeting only courses that are actually named "Physics".  They're the only ones that we should use the SVG background for.  The others can have the colors if the `catalog_offering_identifier` is known, but we don't want to use an SVG that has an incorrect title.